easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.19-foss-2023b-R-4.4.1.eb

```diff
@@ -0,0 +1,1365 @@
+    }),
+    ('EBImage', '4.46.0', {
+        'checksums': ['5c343d8f0438bc9e1d317e9d8f1300ef4af86e1a8f1c783d9099d34493754a61'],
+    }),
+    ('GenomicScores', '2.16.0', {
+        'checksums': ['2a9e929c00706bdd18741413304f4bed0107b1dc26cb961363a14a91595c4a87'],
+    }),
+    ('BSgenome.Mmusculus.UCSC.mm10', '1.4.3', {
+        'checksums': ['dcf602bf9fadf1ef4ce70e0c0fc92b6435467df5affe7d0872d88a93b99ff9ee'],
+    }),
+    ('TxDb.Mmusculus.UCSC.mm10.knownGene', '3.10.0', {
+        'checksums': ['696281749d01737c94894564d62093433045bc007a4528cc3d94f205edb54977'],
+    }),
+    ('regioneR', '1.36.0', {
+        'checksums': ['2bef2e8c0d08e9bc092123a485aec22d7e1fe21de81c221aaec5040737a6edf9'],
+    }),
+    ('InteractionSet', '1.32.0', {
+        'checksums': ['1a06acb7baab7846de9fa437c0f9634453ba2a86495eefab9eb0e6673eac7346'],
+    }),
+    ('universalmotif', '1.22.0', {
+        'checksums': ['fed7c59661771c859e093f9a51668468df773e1af0e3e40abec85c3a56ac5acd'],
+    }),
+    ('ChIPpeakAnno', '3.38.0', {
+        'checksums': ['f4e5b298a7c8a6df79c1a02291aecf20bcba400a4e343541e8a3f790de28f7bc'],
+    }),
+    ('seqLogo', '1.70.0', {
+        'checksums': ['397e7f06b2c5074eb53d4370bf4d13678cec296a844b3e378ada6a13c91b8c94'],
+    }),
+    ('rGADEM', '2.52.0', {
+        'checksums': ['cace9996ff0b67fa5b5b640e5eac1e11ecd1c9eeb32e4f0e089edc20bb753bb4'],
+    }),
+    ('MotifDb', '1.46.0', {
+        'checksums': ['707d54cd3bee0064c8ff5ea473cf98f4a5805d0d20a4977e2fc200f49e391ed5'],
+    }),
+    ('poweRlaw', '0.80.0', {
+        'checksums': ['713556af1f47e1de749670d08f963688908cfa80e9dfda590efd1a28441772cb'],
+    }),
+    ('CNEr', '1.40.0', {
+        'checksums': ['5b1b172b23847768ff56f1d823634b02797ccf0ad1487df9b261d18fee816ea6'],
+    }),
+    ('DirichletMultinomial', '1.46.0', {
+        'checksums': ['0b7f0d48ac584d1f50ab89ab928b7ef40cc577c6d317514d9653136737414d36'],
+    }),
+    ('TFMPvalue', '0.0.9', {
+        'checksums': ['b9db56e75e2cee840d8b7861686dec07ee2c40cbc7d55361e5d04f1bf0c65de7'],
+    }),
+    ('TFBSTools', '1.42.0', {
+        'checksums': ['505bc906a725f0093ce833f06a29ef317cb2818e737aea725ba984d3ebd3b52e'],
+    }),
+    ('motifStack', '1.48.0', {
+        'checksums': ['72ea38e4e05695e4bb92433a6d0a61868a7856623f5f15026191bc0954f86497'],
+    }),
+    ('ATACseqQC', '1.28.0', {
+        'checksums': ['beea902891ff101d81bb71a2f00626b864b948ce446f0b81aab41b08ff6d2c52'],
+    }),
+    ('ResidualMatrix', '1.14.0', {
+        'checksums': ['f63df941d0df726562ead6d3b8ff87e65c728cbaf6ca4ea615028353e8ff47e0'],
+    }),
+    ('batchelor', '1.20.0', {
+        'checksums': ['80661ba295afddc1cd9b95a7b97994d71a979a07018f3f936c36840341cde0c0'],
+    }),
+    ('gsmoothr', '0.1.7', {
+        'checksums': ['b75ffd2a4a0f357762e02e46e355b45cc90ea637830f0a1b01f216bb4541e903'],
+    }),
+    ('Ringo', '1.66.0', {
+        'checksums': ['adc7f824026aff62b36a71827ef815e11d4cb92a94762e0320248649da5400b2'],
+    }),
```

Comment:
Ringo was removed in 3.19 and is no longer available at the usual URL, although it is available at https://bioconductor.org/packages/3.18/bioc/src/contrib/Ringo_1.66.0.tar.gz.

See https://bioconductor.org/about/removed-packages/

```suggestion
```